index.js

```diff
@@ -9,15 +9,17 @@ module.exports = function (pid, cb) {
       return p.PID
     })
 
-    pidusage.stat(pids, function(err, stats) {
+    pids.push(pid)
```

Comment:
does this make sense @simonepri ?

Reply:
Absolutely I was thinking about that.
But i would push it on the top instead.

Reply:
you mean unshift? :P

Reply:
Yes I mean unshit. Anyway let me finish the work on pidusage.
Then I will dive on that.